packages/react/src/River/River.module.css

```
@@ -111,3 +112,71 @@
     margin-top: var(--base-size-48);
   }
 }
+/* end River */
+
+/* RiverBreakout */
+.RiverBreakout {
+  display: flex;
+  flex-direction: column;
+  gap: var(--brand-River-spacing-inner);
+  padding-block-end: var(--brand-River-spacing-outer);
+  padding-block-start: var(--brand-River-spacing-outer);
+}
+  
+.RiverBreakout .River__text {
+  color: var(--brand-color-text-default);
+  font-size: var(--brand-text-size-600);
+  font-weight: var(--base-text-weight-semibold);
+  letter-spacing: var(--brand-heading-letterSpacing-600);
+  line-height: var(--brand-text-lineHeight-600);
```

After you pull `main`, these will need to change to use 500 instead..

```suggestion
  font-size: var(--brand-text-size-500);
  font-weight: var(--brand-heading-weight-500);
  letter-spacing: var(--brand-heading-letterSpacing-500);
  line-height: var(--brand-text-lineHeight-500);
```